src/features/productManage/provider/PetProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, useContext, ReactNode } from "react"
+
+export interface Card {
+  name: string
+  type: string
+  description: string
+  isAdopt: boolean
+  updatedAt: string
+}
+
+const CardsContext = createContext<Card[] | undefined>(undefined)
+
+interface CardsProviderProps {
+  children: ReactNode
+  cards: Card[]
+}
+
+export const CardsProvider = ({ children, cards }: CardsProviderProps) => {
+  return <CardsContext.Provider value={cards}>{children}</CardsContext.Provider>
+}
+
+export const useCards = () => {
```

Comment:
No need to use context since it is only one level parsing from CPage -> PetList (via Context) and PetList -> Card (via Params)

Reply:
Fixed